src/dmd/hdrgen.h

```diff
@@ -19,7 +19,7 @@ class Module;
 void genhdrfile(Module *m);
 void moduleToBuffer(OutBuffer *buf, Module *m);
 
-const char *parametersTypeToChars(Parameters *parameters, int varargs);
+const char *parametersTypeToChars(ParameterList pl);
```

Comment:
May need to have an overload depending on ABI issues, but otherwise looks good.

Reply:
Or even better, make it a ref parameter